cypress/support/fragments/inventory/inventoryInstances.js

```diff
@@ -129,12 +141,13 @@ export default {
             });
           });
       });
+    return instanceId;
   },
 
   deleteInstanceViaApi(itemBarcode) {
     cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${itemBarcode}"` })
       .then((instance) => {
-        cy.deleteItem(instance.items[0].id);
+        instance.items.forEach((item) => cy.deleteItem(item.id));
         cy.deleteHoldingRecord(instance.holdings[0].id);
         cy.deleteInstanceApi(instance.id);
```

Comment:
Please rename delete methods to have the value in name: 'ViaApi'. For example: deleteItem => deleteItemViaAPI

Reply:
I will fix it after merge https://github.com/folio-org/stripes-testing/pull/470

Reply:
done